lerna.json

```diff
@@ -1,5 +1,6 @@
 {
   "npmClient": "yarn",
   "useWorkspaces": true,
-  "version": "independent"
+  "version": "independent",
+  "useNx": true
```

Comment:
@michaldudak Curious how it worked out - are there any benefits of using nx?

Reply:
I haven't noticed any improvements in build times, but we now get a nicer preview of what's being built/checked when running locally.

Reply:
Looking at https://nx.dev/migration/lerna-and-nx#speed-up-lerna-with-nx's-powerful-task-scheduler, I guess there is no performance difference because:

- We are already running all the tasks in parallel
- We run one task, which is the build one, which most often can't use the cache because it's against new sources